thirdeye-integration-tests/src/test/resources/anomalyresolution/config/server.yaml

```diff
@@ -63,6 +63,7 @@ taskDriver:
   noTaskDelay: 1
   randomDelayCap: 1
   taskFailureDelay: 1
+  newAcquisitionLogic: true
```

Comment:
Until we have not rolled out the new flow completely
should we keep some integration tests for old logic as well? 
Otherwise any faulty changes to old logic might not get caught

Reply:
yeah I thought about this. 
the legacy logic really should not be changed anymore, and I'd like to push for the new one aggressively.
The codepaths reached by the old one are pretty obvious and it will be easy to catch if someone (= you or me) edits it. If it's the case we can always run on the old logic by changing this value manually.